src/components/text-area/assets/text-area/t9n/messages_en.json

```diff
@@ -0,0 +1,4 @@
+{
+  "invalid": "Invalid",
+  "overLimit": "Character limit exceeded"
```

Comment:
Suggestion: as a shorter alternative, the native textarea uses `tooLong` as the name for this error. I'll leave this up to you.

Reply:
I prefer using more descriptive error. This suggestion was made during design discussions. WDYT @ashetland ?

Reply:
Are we talking about the name or the message?

Reply:
checked with Franco, it is regarding the name. Sorry for the confusion. I think prop name can be changed to `tooLong`.